public/js/pimcore/startup.js

```diff
@@ -822,6 +821,10 @@ Ext.onReady(function () {
                         }
                     }
                     break;
+                default:
+                    if (!treeConfig.hidden) {
```

Comment:
Should we really check for `hidden` property? or simply add it to the custom trees?

Reply:
@dvesh3 we should check for hidden property before adding it to custom trees 
 if this Perspective element tree is hidden so why i will add it in the custom trees and add it to our event and then check is hidden or not after I listen to this event , and in all cases it will not create tree to this perspective element , so i didn't add it to custom trees array

Reply:
I think the event listener should decide what to do with hidden trees. It might be required in some cases that you deal with hidden trees and add them to tree panel on some actions.